### PR TITLE
Fixes #22870: Use 8080 port to talk with the webapp in hooks

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/22870

### CODE REVIEW

**Comment by amousset** [comment]
PR updated with a new commit

File: webapp/sources/rudder/rudder-core/src/main/resources/hooks.d/policy-generation-finished/60-trigger-node-update [hunk 1]
```diff
@@ -88,7 +88,7 @@ if [ -n "${RUDDER_NODE_IDS[0]}" ] && (( ${TO_UPDATE} > 0 )) ; then
   if [ -n "${TEST}" ]; then
     echo "${NODES}"
   else
-    curl -X POST 'http://localhost/rudder/relay-api/remote-run/nodes' -d "asynchronous=true" -d "keep_output=false" -d @- <<CURL_DATA
+    curl -X POST 'http://localhost:3030/rudder/relay-api/remote-run/nodes' -d "asynchronous=true" -d "keep_output=false" -d @- <<CURL_DATA
```
Comment (ncharles) [inline webapp/sources/rudder/rudder-core/src/main/resources/hooks.d/policy-generation-finished/60-trigger-node-update:91]: wow, didn't see it coming

**Review by ncharles — APPROVED** [review]
Thank you SO much

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**